### PR TITLE
Gmake fixes

### PR DESCRIPTION
**Pull request by tvandijck**
We ran into a couple of gmake backend issues with defines and precompiled header files.

### CODE REVIEW

File: src/actions/make/make_cpp.lua [hunk 1]
```diff
@@ -481,15 +481,26 @@
...
+				if os.isfile(testname) then
+					pch = project.getrelative(cfg.project, testname)
+					found = true
+					break
+				end
 			end
 		end
+
 		if not found then
+			print('not found', pch)
```
Comment (starkos) [inline src/actions/make/make_cpp.lua]: This should go away, but maybe use a `p.warnOnce()`?

**Comment by starkos** [comment]
One comment inline, otherwise LGTM!

File: src/actions/make/make_cpp.lua (hunk 1 above)
Comment (tvandijck) [inline src/actions/make/make_cpp.lua]: actually that should indeed go away... a leftover from me trying to figure out why it wasn't finding my pch files.

**Comment by tvandijck** [comment]
fixed

**Comment by samsinsane** [comment]
LGTM.

**Pull request merged**